\hypertarget{classnumpy_1_1lib_1_1__iotools_1_1ConverterError}{}\section{numpy.\+lib.\+\_\+iotools.\+Converter\+Error Class Reference}
\label{classnumpy_1_1lib_1_1__iotools_1_1ConverterError}\index{numpy.\+lib.\+\_\+iotools.\+Converter\+Error@{numpy.\+lib.\+\_\+iotools.\+Converter\+Error}}


Inheritance diagram for numpy.\+lib.\+\_\+iotools.\+Converter\+Error\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=221pt]{classnumpy_1_1lib_1_1__iotools_1_1ConverterError__inherit__graph}
\end{center}
\end{figure}


Collaboration diagram for numpy.\+lib.\+\_\+iotools.\+Converter\+Error\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=221pt]{classnumpy_1_1lib_1_1__iotools_1_1ConverterError__coll__graph}
\end{center}
\end{figure}


\subsection{Detailed Description}
\begin{DoxyVerb}Exception raised when an error occurs in a converter for string values.\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/lib/python3.\+9/site-\/packages/numpy/lib/\hyperlink{__iotools_8py}{\+\_\+iotools.\+py}\end{DoxyCompactItemize}
